client/wfnews-war/src/main/angular/src/app/components/report-of-fire/disclaimer-page/rof-disclaimer-page.component.ts

```diff
@@ -10,11 +12,43 @@ import { ReportOfFire } from '../reportOfFireModel';
 export class RoFDisclaimerPage extends RoFPage {
   public imageUrl: string
   public message: string;
+  public currentBrowser: string;
+  public hideButtons : boolean;
+
+  public constructor(
+    private commonUtilityService: CommonUtilityService,
+    ) {
+    super();
+  }
 
   initialize (data: any, index: number, reportOfFire: ReportOfFire) {
     super.initialize(data, index, reportOfFire);
     this.imageUrl = data.imageUrl;
     this.message = data.message.split('\n');
-    
+    const userAgent = window.navigator.userAgent.toLowerCase();
+
+    // Check for the presence of certain keywords in the user agent to determine the browser
+    if (userAgent.includes('chrome')) {
```

Comment:
Could be a switch vs. a chain of if-else

Reply:
thanks ! modified